addons/website_slides/static/src/js/slides_course_quiz_finish.js

```diff
@@ -0,0 +1,148 @@
+odoo.define('website_slides.quiz.finish', function (require) {
+'use strict';
+
+var Dialog = require('web.Dialog');
+var core = require('web.core');
+var _t = core._t;
+
+var SlideQuizFinishModal = Dialog.extend({
```

Comment:
And maybe one tiny docstring here?

"This modal is used when the user finishes the quiz blablabla
It handles the animation of xp gain and leveling up blablabla"